canonical-websites/www.ubuntu.com
# Update IBM Z download page

## Done

- Updated the list of drawers
- Updated the case on Z13 to z13
- Removed z systems to IBM Z
- Updated the copy doc

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/download/server/s390x](http://0.0.0.0:8001/download/server/s390x)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1d85IfqqxZYYZ0orj5lBhqLWLGpLYWRLdvs75lYsplGs/edit#)

## Issue / Card

Fixes #4664
